browser/src/Services/Configuration/DefaultConfiguration.ts

```diff
@@ -115,40 +115,126 @@ const BaseConfiguration: IConfigurationValues = {
     "editor.cursorColumn": false,
     "editor.cursorColumnOpacity": 0.1,
 
-    "editor.tokenColors": [
-        {
-            scope: "variable.object",
-            settings: "Identifier",
+    "editor.tokenColors": {
+        "variable.object": {
+            scope: [],
+            settings: {
+                foreground: null,
+                background: null,
+                bold: null,
+                italic: null,
+            },
         },
-        {
-            scope: "variable.other.constant",
-            settings: "Constant",
+        "support.class": {
+            scope: ["support.class.builtin", "support.type.primitive"],
+            settings: {
+                foreground: null,
+                background: null,
+                bold: null,
+                italic: null,
+            },
         },
-        {
-            scope: "variable.language",
-            settings: "Identifier",
+        "support.variable": {
+            scope: ["support.variable.property.dom"],
+            settings: {
+                foreground: null,
+                background: null,
+                bold: null,
+                italic: null,
+            },
         },
-        {
-            scope: "variable.parameter",
-            settings: "Identifier",
+        "meta.object": {
+            scope: ["meta.object.type"],
```

Comment:
Hmm, I don't understand how the `meta.object` relates to the scope here.  The previous scheme seemed simpler to me - and consistent with how the `tokenColors` in a theme is defined.

Reply:
haha as is always the way I skim read the docs but went through them a bit more closely now and not haphazardly applying styles to scopes against what the documentation recommends actually really improves highlighting 😆